conformance_pack/cloudtrail.sp

```diff
@@ -577,6 +587,29 @@ query "cloudtrail_multi_region_read_write_enabled" {
   EOQ
 }
 
+query "cloudtrail_trail_insight_selectors_and_logging_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when not is_logging  then 'skip'
```

Comment:
Shouldn't the control go to an alarm if logging is disabled?

As per the description - `CloudTrail trails should have insight selectors and logging enabled`, I think we should fail the control if it does not have logging enabled.